en/skills/11. Tables.ipynb

```diff
@@ -0,0 +1,458 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "a4a92198",
+      "metadata": {
+        "id": "a4a92198"
+      },
+      "source": [
+        "# Working with Tables\n",
+        "\n",
+        "Tables are a simple way to organize and store data. They are commonly used in spreadsheets (think Excel or Google Sheets) and databases, where each row holds a record, and each column holds a different attribute of that record.\n",
+        "\n",
+        "CSV files (Comma-Separated Values) are one of the most popular formats for storing tables. This files can be easily exported from an Excel or Google Sheets file (or opened via one of these applications)."
```

Comment:
Generally for the skill lessons I try to provide a concrete example. In this case, I think it would make sense to have an example CSV (something straightforward) that we provide among the corpora, then demonstrate loading it and viewing it (before we do anything with their own files)

Reply:
I have this file, but I'm not sure how to upload it to the site or you mean in a different way? It's the data.csv used in the reading example. Let me know how and I'll add it.